### DISCUSSION TITLE
Non ->TikTok returned an empty response

### DISCUSSION THREAD

**Discussion by negi20001211** [davidteather/TikTok-Api#1187] [General]
For the past two weeks or so, I've been receiving empty responses for all requests.
Is there anyone with the same symptoms? ?

**Comment by zakrian07** [comment]
I’m getting same issue

**Comment by PyBoba** [comment]
found a solution to the problem?

**Comment by IntegerOverfl0w** [comment]
This happens when you need a captcha to continue.
Open the browser by passing `headless=False` to `create_sessions`, click a video and refresh the page and you should get a captcha.
Complete the captcha and you should start getting results again

**Comment by SanclerMoonshot** [comment]
I was getting the same issue, and it stopped when I passed my `ms_token` to the `create_sessions`
```
await api.create_sessions(ms_tokens=[ms_token], num_sessions=1, sleep_after=3)
```

I tried the `headless=False` but it didn't work for me.
